one console: replace cluster with instance for Starter and Essential (#22542)
pingcap/docs: develop/dev-guide-aws-appflow-integration.md

    ```

5. After the `sf_account` table is created, click **Connect**. A connection dialog is displayed.

> 6. In the **Connect to TiDB-Connector** dialog, enter the connection properties of the {{{ .starter }}} instance. If you use a {{{ .starter }}} instance, you need to set the **TLS** option to `Yes`, which lets the TiDB connector use the TLS connection. Then, click **Connect**.

Author: ```suggestion
6. In the **Connect to TiDB-Connector** dialog, enter the connection properties of the {{{ .starter }}} instance. For {{{ .starter }}}, you need to set the **TLS** option to `Yes`, which lets the TiDB connector use the TLS connection. Then, click **Connect**.
```
